### DISCUSSION TITLE
how to  serialize uniformly in axios.get?

### DISCUSSION THREAD

**Discussion by zhoufanglu** [axios/axios#5874] [Q&A]
### Describe the bug

![image](https://user-images.githubusercontent.com/24451091/212629537-46b2cfd6-04d2-407a-826c-80deb097524b.png)

### To Reproduce

axios.get 

### Code snippet

```js
testC(params: any) {
    return axios.get(
      `/testC?${qs.stringify(params, { arrayFormat: 'repeat' })}`
    )
  },
  testD(params: any) {
    return axios.get(
      `/testD?${qs.stringify(params, { arrayFormat: 'repeat' })}`
    )
  }
```

### Expected behavior

-

### Axios Version

"axios": "^1.1.3"

### Adapter Version

-

### Browser

chrome

### Browser Version

-

### Node.js Version

14

### OS

-

### Additional Library Versions

```bash
-
```

**Comment by MahirMahdi** [comment]
It's hard to say without more context. Assuming you have installed and imported all the necessary packages, I guess params is not in the expected format or is not valid.

**Comment by jasonsaayman** [comment]
Closed no context added also very stale
